code/modules/mob/living/silicon/examine.dm

```diff
@@ -4,3 +4,16 @@
 		. += "<b>[src] has the following laws:</b>"
 		for(var/law in laws.get_law_list(include_zeroth = TRUE))
 			. += law
+
+	// BEGIN SKYRAT EDIT: OOC Notes for sillicons
+	if(client)
+		var/line
+		if(length(client.prefs.features["silicon_flavor_text"]))
+			var/message = client.prefs.features["silicon_flavor_text"]
+			if(length_char(message) <= 40)
+				line = "<span class='notice'>[message]</span>"
+			else
+				line = "<span class='notice'>[copytext_char(message, 1, 37)]... <a href='?src=[REF(src)];lookup_info=silicon_flavor_text'>More...</a></span>"
+		line += " <span class='notice'><a href='?src=[REF(src)];lookup_info=ooc_prefs'>\[OOC\]</a></span>"
+		. += line
+	// END SKYRAT EDIT
```

Comment:
```suggestion
	//SKYRAT EDIT ADDITION BEGIN - Customization
	var/line
if(length(client.prefs.features["silicon_flavor_text"]))
		var/message = client.prefs.features["silicon_flavor_text"]
		if(length_char(message) <= 40)
			line = "<span class='notice'>[message]</span>"
		else
			line = "<span class='notice'>[copytext_char(message, 1, 37)]... <a href='?src=[REF(src)];lookup_info=silicon_flavor_text'>More...</a></span>"
	if(client)
		line += " <span class='notice'><a href='?src=[REF(src)];lookup_info=ooc_prefs'>\[OOC\]</a></span>"
	. += line
	//SKYRAT EDIT ADDITION END
```

Reply:
Man I hope I didnt mess up any indentitation. but basically the point is to show OOC prefs only when a client is present, but flavor text always.